Service/XxllncToZGWZaakTypeService.php

```diff
@@ -0,0 +1,376 @@
+<?php
+
+namespace CommonGateway\XxllncZGWBundle\Service;
+
+use App\Entity\Entity as Schema;
+use App\Entity\Gateway as Source;
+use App\Entity\Mapping;
+use App\Entity\ObjectEntity;
+use App\Service\SynchronizationService;
+use CommonGateway\CoreBundle\Service\CallService;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\Persistence\ObjectRepository;
+use Exception;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class XxllncToZGWZaakTypeService
+{
+    private EntityManagerInterface $entityManager;
+    private CallService $callService;
+    private SynchronizationService $synchronizationService;
+    private SymfonyStyle $io;
+    private array $configuration;
+    private array $data;
+
+    private ?ObjectRepository $objectRepo;
+    private ?ObjectRepository $schemaRepo;
+    private ?ObjectRepository $sourceRepo;
+    private ?ObjectRepository $mappingRepo;
+
+    private ?Source $xxllncAPI;
+    private ?Schema $zaakTypeSchema;
+    private ?Schema $rolTypeSchema;
+    private ?Mapping $caseTypeMapping;
+
+    private ?ObjectEntity $catalogusObject;
+
+    private array $skeletonIn;
+
+    /**
+     * @param EntityManagerInterface $entityManager
+     */
+    public function __construct(
+        EntityManagerInterface $entityManager,
+        SynchronizationService $synchronizationService,
+        CallService $callService
+    ) {
+        $this->entityManager = $entityManager;
+        $this->synchronizationService = $synchronizationService;
+        $this->callService = $callService;
+
+        $this->objectRepo = $this->entityManager->getRepository(ObjectEntity::class);
+        $this->schemaRepo = $this->entityManager->getRepository(Schema::class);
+        $this->sourceRepo = $this->entityManager->getRepository(Source::class);
+        $this->mappingRepo = $this->entityManager->getRepository(Mapping::class);
+
+        $this->skeletonIn = [
+            'handelingInitiator'   => 'indienen',
+            'beginGeldigheid'      => '1970-01-01',
+            'versieDatum'          => '1970-01-01',
+            'doel'                 => 'Overzicht hebben van de bezoekers die aanwezig zijn',
+            'versiedatum'          => '1970-01-01',
+            'handelingBehandelaar' => 'Hoofd beveiliging',
+            'aanleiding'           => 'Er is een afspraak gemaakt met een (niet) natuurlijk persoon',
+        ];
```

Comment:
skeletonIn feels like something we should configure in some .json file and not in a service variable or constructor

Reply:
Should be in the mapping
Add a todo for this

Reply:
![image](https://user-images.githubusercontent.com/57346398/224293107-b892ce26-6ee5-46a5-9946-3e03c9d6bf8c.png)
Git loopt achter